### PR TITLE
IX Bid Adapter: send bid floor per size in format ext

### PR DESCRIPTION
**Pull request by shahinrahbariasl**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- Start sending bid floor per size to Exchange. This is to later support floors for multi size request.

For more information contact shahin.rahbariasl@indexexchange.com

### CODE REVIEW

**Comment by patmmccann** [comment]
please title your prs on your module with your module name

**Comment by shahinrahbariasl** [comment]
> please title your prs on your module with your module name

Hey Pat, thank you for making the title change. Was going to mark the PR as ready for review and update it but thanks!

File: modules/ixBidAdapter.js [hunk 1]
```diff
@@ -762,10 +762,15 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
...
-          delete bannerFormat.ext.sid;
+      for (let i = 0; i < _bannerImpression.banner.format.length; i++) {
+        // We add sid in imp.ext.sid therefore, remove from banner.format[].ext
+        if (_bannerImpression.banner.format[i].ext != null && _bannerImpression.banner.format[i].ext.sid != null) {
+          delete _bannerImpression.banner.format[i].ext.sid;
+        }
+
+        // add floor per size
+        if ('bidfloor' in impressionObjects[i]) {
+          _bannerImpression.banner.format[i].ext.bidfloor = impressionObjects[i].bidfloor
```
[the remark this answers is not in the document]
Comment (shahinrahbariasl) [reply modules/ixBidAdapter.js:773]: Hey Pat, not for now. Currency will be the same for all sizes. We're only going to add support for floor per size for now.

[the remark this answers is not in the document]
Comment (lksharma) [reply modules/ixBidAdapter.js:773]: @patmmccann Is there anything else you need from our end before merging?

**Review by patmmccann — APPROVED** [review]

File: modules/ixBidAdapter.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (patmmccann) [reply modules/ixBidAdapter.js:773]: nope looks good!

**Pull request merged**